shell/client/shell.html

```diff
@@ -1094,7 +1094,7 @@ <h4 class="share-with-others">Share with others</h4>
       </span>
 
       <label>Relay port:
-        <input type="text" name="smtpPort" value="{{smtp.port}}" />
+        <input type="text" name="smtpPort" value="{{smtp.port}}" pattern="[0-9]+" />
```

Comment:
`type="range"` produces a slider, not a text input box, so I think that's not what we want.

`type="number"` is mildly odd in that it gives up/down arrows, but it does provide the mobile hint and better validation behavior, so I've gone with that.